### DISCUSSION TITLE
为什么没有新版的optifine了

### DISCUSSION THREAD

**Discussion by usdasus** [Meloong-Git/PCL#4565] [提问] [answered]
为什么没有新版的optifine了，在网上都可以搜到optifine适配MC新版本的版本为什么龙猫这里没有？

**Comment by hejiehao** [comment]
？截个图看看

**Comment by zkitefly** [comment, marked as answer]
> 在网上都可以搜到optifine适配MC新版本的版本

是指 OptiKai？

官方版本目前定在 1.20.6

![image](https://github.com/user-attachments/assets/12b10a1d-6a5d-4157-9986-5c8672173bb2)

**Comment by bmmNbhjd** [comment]
推荐去找 sp614x 问问
